tasks/install_shell.sh

```diff
@@ -483,6 +483,18 @@ case $platform in
     filename="${collection}-release-${deb_codename}.deb"
     download_url="${apt_source}/${filename}"
     ;;
+  "Linuxmint"|"LinuxMint")
+    info "Mint platform! Lets get you a DEB..."
+    case $major_version in
+      "3")  deb_codename="stretch";;
```

Comment:
I do not have a way to test this, is "3" what the $major_version will resolve to? What version of the `facts` module are you using?

Reply:
3 is LDME
18, 19 is LinuxMint